src/Interpreters/Context.cpp

```diff
@@ -2067,20 +2072,48 @@ Lemmatizers & Context::getLemmatizers() const
 }
 #endif
 
-BackupsWorker & Context::getBackupsWorker() const
+void Context::initializeBackupsWorker()
```

Comment:
Could we move most of this function's body to `BackupsWorker.cpp`? Because `Context.cpp` is already quite a long file.

Reply:
Moved.